src/components/search/results/results.vue

```diff
@@ -3,7 +3,7 @@
     <BannersList>
       <PromotedsList>
         <NextQueriesList>
-          <component :is="grid" :columns="2" class="x-grid" :animation="staggeredFadeAndSlide">
+          <BaseVariableColumnGrid class="x-grid" :animation="staggeredFadeAndSlide">
```

Comment:
Why are you adding this change? I mean, you are configuring the toolbar

Reply:
I added that change because now the number of columns in mobile can be changed.
If I wouldn't change this component, the ColumnPicker in mobile would not have worked.

Reply:
Ah yes, you are fucking right, sorry.